src/sentry/templates/sentry/emails/identity-invalid.txt

```diff
@@ -0,0 +1,8 @@
+Action Required
+---------------
+
+An identity with a third party service provider ({{ identity.provider }}) failed to authenticate and has been removed.
+
+You will need to associate your account with {{ identity.provider }} to continue using the integration.
```

Comment:
Plain text doesn't have the message about fetching commits?

Reply:
yeah i didnt wanna deal w/ formatting since its such an uncommon case